provisioners/puppet/modules/config/data/RedHat/Amazon-2.yaml

```diff
@@ -17,16 +17,16 @@ config::package_manager_packages:
   - name: inspec
     version: 1.51.6
     provider: puppet_gem
+  - name: python.x86_64
+  - name: python2-pip.noarch
+  - name: python2-cheetah.x86_64
```

Comment:
Do we need those python2 packages ?

Reply:
Same for the other OS

Reply:
Did we want to keep them for backwards compatibility?

Reply:
Removed